pkg/test/env_test.go

```diff
@@ -138,7 +138,7 @@ func TestUnsetEnvVarAndRestore(t *testing.T) {
 }
 
 func unsetVariable(t *testing.T) string {
-	u, err := uuid.NewV4()
+	u, err := uuid.NewRandom()
 	require.NoError(t, err)
 	varName := u.String()
```

Comment:
this whole thing could be simplified to:
```suggestion
	varName := uuid.NewString()
```
right?

Reply:
yep, done in 96f0276